lib/jormungandr/src/Cardano/Wallet/Jormungandr/Binary.hs

```diff
@@ -372,6 +386,44 @@ putTx inputs outputs = do
         putAddress address
         putWord64be $ getCoin coin
 
+{-------------------------------------------------------------------------------
+                                 Stake Distribution
+-------------------------------------------------------------------------------}
+
+data StakeApiResponse = StakeApiResponse
+    { epoch :: Word64
+    , stake :: StakeDistribution
+    } deriving (Show, Eq, Generic)
+
+-- | Coins are stored as Lovelace (reminder: 1 Lovelace = 1e-6 ADA)
+newtype Stake = Stake { getStake :: Word64 } deriving (Show, Eq, Generic)
```

Comment:
I am thinking if `Stake` and `StakeDistribution` should not be in `Cardano.Wallet.Primitive.Types` :thinking:  @KtorZ @jonathanknowles  what do you think?

Reply:
ok, decided to move `Stake` and `StakeDistribution` to `Cardano.Wallet.Primitive.Types`